eu.skribisto.skribisto.yml

```diff
@@ -0,0 +1,37 @@
+app-id: eu.skribisto.skribisto
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: skribisto
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+separate-locales: false
```

Comment:
```suggestion
```

Locales are split into separate ref and only these requested by the OS are installed. For example, if I were using pl_PL, and the app is translated to Polish, flatpak will download only that language.

Reply:
Thank you for the explanation. Maybe I understood wrongly. Are we talking about the translations of my software ? My compiled translations (Qt's .qm files) are included in the executable.

Reply:
Looks like it's not needed here also in your case as flatpak-builder detects there's no locales to split but let's keep it explicit.